.github/workflows/run_tests.yml

```diff
@@ -3,10 +3,11 @@ on:
   push:
     branches:
       - main
+      - restructure-repository
```

Comment:
????

Reply:
Thanks. It's a remnant from developing on my fork. I'll remove it.

Reply:
Resolved: ca05faa81f3bf07a7e62c2a2458a5764374006c2